src/org/zalando/stups/essentials/api.clj

```diff
@@ -168,14 +168,17 @@
             400
             "A resource-owner-scope requires its resource type to have at least one resource owner"
             {:resource_type_id resource_type_id :scope_id scope_id}))
+        (let [defaults {:criticality_level 2}
+              scope-keys (select-keys scope [:summary
+                                             :description
+                                             :user_information
+                                             :is_resource_owner_scope
+                                             :criticality_level])
+              scope-ids {:resource_type_id resource_type_id
+                         :scope_id         scope_id}]
         (sql/cmd-create-or-update-scope!
-          {:resource_type_id        resource_type_id
-           :scope_id                scope_id
-           :summary                 (:summary scope)
-           :description             (:description scope)
-           :user_information        (:user_information scope)
-           :is_resource_owner_scope (:is_resource_owner_scope scope)}
-          {:connection db})
+          (merge-with #(or %2 %1) defaults scope-keys scope-ids)
```

Comment:
Das Erste Argument von "merge-with" ist eine Funktion, mit der man die Kontrolle übernehmen kann, wie gemerged werden soll. "merge" überschreibt die Werte in der map, auch wenn sie nil sind.
=> (merge {:a 1 :b 2} {:b nil})
{:b nil, :a 1}
=> (merge-with #(or %2 %1) {:a 1 :b 2} {:b nil})
{:b 2, :a 1}